doc/development-context-aware-applications/v2/how-to-update-and-query-context-information.md

```diff
@@ -0,0 +1,243 @@
+      "reviewBody": "Cheap and nice place to eat.",
+      "ratingValue": 4
+    }
+
+Each time a new Review entity is created, the average rating for the
+corresponding restaurant is recalculated by the application backend
+which (playing also the role of Context Producer) **updates** the
+Restaurant entity accordingly:
+
+    PUT <cb_host>:<cb_port>/v2/entities/0115206c51f60b48b77e4c937835795c33bb953f/attrs/aggregateRating/value
+    {
+      "reviewCount": 2,
+      "ratingValue": 4
+    }
+
+Also, the user can get the information of a given Restaurant using
+the smartphone application. In that case the application works as
+Context Consumer, **querying** the Restaurant entity. For example, to get
+the aggregateRating attribute, the client application could query for
+it in the following way:
+
+    GET <cb_host>:<cb_port>/v2/entities/0115206c51f60b48b77e4c937835795c33bb953f/attrs/aggregateRating/value
+    {
+      "reviewCount": 2,
+      "ratingValue": 4
+    }
+
+You can also obtain the values of all attributes of the "Elizalde"
+restaurant in a single shot:
+
+    GET <cb_host>:<cb_port>/v2/entities/0115206c51f60b48b77e4c937835795c33bb953f
+    {
+      "id": "0115206c51f60b48b77e4c937835795c33bb953f",
+      "type": "Restaurant",
+      "address": {
+        "type": "PostalAddress",
+        "value": {
+          "streetAddress": "Cuesta de las Cabras Aldapa 2",
+          "addressRegion": "Araba",
+          "addressLocality": "Alegría-Dulantzi",
+          "postalCode": "01240"
+        },
+        "metadata": {}
+      },
+      "aggregateRating": {
+        "type": "none",
+        "value": {
+          "reviewCount": 2,
+          "ratingValue": 4
+        },
+        "metadata": {}
+      },
+      "capacity": {
+        "type": "PropertyValue",
+        "value": 50,
+        "metadata": {}
+      },
+      "department": {
+        "type": "none",
+        "value": "Franchise1",
+        "metadata": {}
+      },
+      "description": {
+        "type": "none",
+        "value": "Restaurante de estilo sidrería ubicado en Alegria-Dulantzi. Además ...",
+        "metadata": {}
+      },
+      "location": {
+        "type": "geo:point",
+        "value": "42.8404625, -2.5123277",
+        "metadata": {}
+      },
+      "name": {
+        "type": "none",
+        "value": "Elizalde",
+        "metadata": {}
+      },
+      "occupancyLevels": {
+        "type": "PropertyValue",
+        "value": 0,
+        "metadata": {
+          "timestamp": {
+            "type": "date",
+            "value": "2016-04-20T18:15:15.434Z"
+          }
+        }
+      },
+      "priceRange": {
+        "type": "none",
+        "value": 0,
+        "metadata": {}
+      },
+      "telephone": {
+        "type": "none",
+        "value": "945 400 868",
+        "metadata": {}
+      }
+    }
+
+
+
+Alternatively, if you want to get the **attributes as a key values**, you can use the `keyValues` parameter, in the
```

Comment:
instead of saying 'attributes as a key values' I would say "If you want to get only attribute values"